sbi/simulators/simutils.py

```diff
@@ -86,32 +484,30 @@ def simulate_in_batches(
     num_samples: int,
     simulation_batch_size: int,
     x_dim: torch.Size,
-) -> (torch.Tensor, torch.Tensor):
+) -> (Tensor, Tensor):
     """
-    Draw `num_samples` parameter sets and simulate them in batches of size `simulation_batch_size`.
+    Draw `num_samples` parameter sets and simulate them in batches 
+    of size `simulation_batch_size`.
```

Comment:
I sense a potential confusion between the batch_size of the simulator and the simulation_batch_size?

Reply:
Forget it, it's not very clear in my head. Will raise it online if it bugs me again.